Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDelegate.swift

```diff
@@ -227,5 +227,27 @@ extension FunctionalTableData {
 			let cellConfig = data.sections[indexPath]
 			return cellConfig?.actions.trailingActionConfiguration?.asSwipeActionsConfiguration()
 		}
+		
+		@available(iOS 13.0, *)
+		public func tableView(_ tableView: UITableView, contextMenuConfigurationForRowAt indexPath: IndexPath, point: CGPoint) -> UIContextMenuConfiguration? {
+			let section = data.sections[indexPath.section]
+			let row = section.rows[indexPath.row]
```

Comment:
Same question as for `UICollectionView`. Just pointing it out in case it's worth a check if those indices exist first as I don't have enough context to know if `data` will always contain items at the index paths invoked here